src/App.tsx

```diff
@@ -1,14 +1,53 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [loadOn, setLoadOn] = useState(false);
+  const [todos, setToDos] = useState<Todo[]>([]);
+  const [modalOn, setmodalOn] = useState(false);
+  const [modalToDo, setmodalTodo] = useState<Todo | null>(null);
+  const [sortType, setSortType] = useState<string>('');
+  const [query, setQuery] = useState<string>('');
```

Comment:
You don't need to make these variables in the "app" component and drill it into the filter. Just create it in the filter component

Reply:
I think it's not critical as we need to pass the filtered todos to TodosList component